portfolio/src/main/webapp/script.js

```diff
@@ -80,3 +80,12 @@ function createCommentElement(text) {
   commentContainer.innerText = text;
   return commentContainer;
 }
+
+/**
+ * Calls post method of delete-data servlet to delete all comments.
+ * Calls getComments() to update the page.
```

Comment:
```suggestion
 * Deletes all comments and refreshes comments displayed on the page.
```
nit: I suggested an update to the docstring for this function. Generally, we don't need to try to describe in detail all the implementation details of _how_ the the function works, but _what_ it does and any impact we should be aware of e.g. reloading comments on the page.

Reply:
Thank you! I will try to focus these on broad goals rather than methods.